docs/maven-plugin.md

```diff
@@ -88,6 +88,8 @@ Parameters:
 | **pmdFilter** | String | Relative path of a suppression.properties file that lists classes and rules to be excluded from failures. If not set no classes and no rules will be excluded |
 | **maven.pmd.version** | String | The version of the maven-pmd-plugin that will be used (Default value is **3.7**)|
 | **pmdPlugins** | List<Dependency> | A list with artifacts that contain additional checks for PMD |
+|**skipPmd**|boolean|If set to true pmd checks will not be executed|
+|**skippedCategories**|Collection<String>|A collection of all categories that will be skipped during the exeuction of PMD. If not set all categories of checks will be executed.|

```

Comment:
Are these categories something existing?

Reply:
Yes, they are part of the official API https://github.com/pmd/pmd/wiki/Rule-Categories